backend/src/main/java/dev/tripdraw/application/TripService.java

```diff
@@ -19,15 +20,15 @@
 public class TripService {
 
     private final TripRepository tripRepository;
-    private final MemberService memberService;
+    private final MemberRepository memberRepository;
 
-    public TripService(TripRepository tripRepository, MemberService memberService) {
+    public TripService(TripRepository tripRepository, MemberRepository memberRepository) {
         this.tripRepository = tripRepository;
-        this.memberService = memberService;
+        this.memberRepository = memberRepository;
     }
 
     public TripCreateResponse create(LoginUser loginUser) {
-        Member member = memberService.validateLoginUser(loginUser);
+        Member member = memberRepository.findByNickname(loginUser.nickname()).get();
```

Comment:
다음과 같이 변경해보는건 어떨까요?

```suggestion
        Member member = memberRepository.findByNickname(loginUser.nickname())
                .orElseThrow(() -> new MemberException(MEMBER_NOT_FOUND));
```

검증된 값이 해당 서비스까지 들어오는 형태라 예외가 발생하지는 않겠지만 
`get()`으로 값을 가져오기 보단 방어적 프로그래밍을 하는게 더 좋아보입니다.

아래의 addPoint 메서드에서도 마찬가지구요 😄

Reply:
```suggestion

    default Member getByNickname(String nickname) {
        return findByNickname(nickname)
                .orElseThrow(() -> new MemberException(MEMBER_NOT_FOUND));
    }
```
이것도 좋아보여요

허브: 나도 이것도 괜찮은 것 같아 다른것도 이렇게 사용하도록 통일한다면?

Reply:
일단은 default 메서드로 사용하는건 나중에 생각해보고 `.orElseThrow(() -> new MemberException(MEMBER_NOT_FOUND));` 를 사용하는것도 좋을 것 같아 👍

Reply:
저도 둘다 좋은 것 같아요!
하나 고르자면 LoginMember를 파라미터로 받는 방식을 고르겠습니다.
(이유 : Controller에서 해당 메서드가 LoginMember가 아닌 nickname만 필요하다는 상세 정보를 모르게 하기 위해)

Reply:
아무래도 고민이었는데 로직상으로 예외가 발생하지 않을 것 같아서 이렇게 해놨습니다.
허브 말대로 방어적으로 프로그래밍 하겠습니다~
말랑도 좋은 의견 고맙습니다!